### PR TITLE
feat: Operate - Website download flow

### PR DESCRIPTION
**Pull request by mohandast52**
- Please use [live preview](https://olas-website-git-mohan-operate-download-flow-autonolas.vercel.app/operate) to review operate
- Pearl-terms [here](https://olas-website-git-mohan-operate-download-flow-autonolas.vercel.app/pearl-terms)

### CODE REVIEW

File: components/OperatePage/GettingStartedIsSimple.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      <div className="flex justify-center items-center gap-8 xs:flex-col">
+        {installSteps.map(({ title }, index) => (
+          <Fragment key={index}>
+            <div className="ml-2">{title}</div>
+            {index !== downloadLinks.length - 1 && <ArrowRight size={16} />}
+          </Fragment>
+        ))}
+      </div>
+
+      <div className="flex justify-center items-center gap-8 xs:flex-col">
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/OperatePage/GettingStartedIsSimple.jsx:60]: you need to use flex-col and then md:flex

[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/OperatePage/GettingStartedIsSimple.jsx]: @mohandast52 `flex justify-center items-center gap-8 xs:flex-col` -> `flex flex-col justify-center items-center gap-8 md:flex-row` try this

File: common-util/Accordion.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            }`}
+          />
+        </div>
+      </button>
+
+      <div
+        className={`grid px-6 bg-white border border-gray-200 overflow-hidden text-slate-600 rounded-xl rounded-t-none ${transition} ${
+          accordionOpen
+            ? 'grid-rows-[1fr] opacity-100 py-4'
+            : 'grid-rows-[0fr] opacity-0'
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply common-util/Accordion.jsx:34]: you need to add common-util to the config `tailwind.config.js`:
```
 darkMode: ['class'],
  content: [
    './pages/**/*.{js,jsx}',
    './components/**/*.{js,jsx}',
    './common-util/**/*.{js,jsx}',
    './app/**/*.{js,jsx}',
    './src/**/*.{js,jsx}',
  ],
```

File: components/OperatePage/MeetTheOperateApp.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    <h2 className={SUB_HEADER_CLASS}>
+      <div className="text-left lg:text-center">
+        Easy setup. Continuous rewards
+      </div>
+    </h2>
+
+    <div className="grid grid-cols-1 gap-8 md:grid-cols-2">
+      {list.map(({ title, desc, icon }) => (
+        <div
+          key={kebabCase(title)}
```
Comment (Tanya-atatakai) [inline components/OperatePage/MeetTheOperateApp.jsx:52]: same question as before, do you think we need to use `kebabCase`

File: components/OperatePage/MeetTheOperateApp.jsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+          className="mb-2 lg:mb-6 w-16 lg:w-24"
+          alt="Operate Logo"
+          src="/images/operate-page/operate-logo.svg"
+          width={96}
+          height={96}
+        />
+
+        <h2 className={SUB_HEADER_CLASS}>Meet the Operate app</h2>
+
+        <div className={DESCRIPTION_CLASS}>
```
Comment (Tanya-atatakai) [inline components/OperatePage/MeetTheOperateApp.jsx:85]: do you think `p` tag will suit here better?

File: components/OperatePage/MeetTheOperateApp.jsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+          Set up and run an agent with just a few clicks, and start earning OLAS
+          tokens by simply keeping your computer on.
+        </div>
+
+        <div className={`${DESCRIPTION_CLASS} mt-6`}>
```
Comment (Tanya-atatakai) [inline components/OperatePage/MeetTheOperateApp.jsx:90]: same here, `div` -> `p`

File: components/OperatePage/WantMoreControl.jsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+          <h2 className={SUB_HEADER_CLASS}>Want more control?</h2>
+
+          <div>
+            {controlList.map((each, index) => (
+              <div key={index} className="py-4 rounded-lg text-gray-600">
+                <div className={`flex items-center mb-1 ${TEXT}`}>
+                  <h2 className="font-semibold">{each.title}</h2>
+                </div>
+
+                <p className="">{each.desc}</p>
```
Comment (Tanya-atatakai) [inline components/OperatePage/WantMoreControl.jsx:40]: maybe we can remove empty className

File: components/Button.jsx [hunk 7]
```diff
@@ -11,74 +33,40 @@ export function Button({
...
+  disabled: PropTypes.bool,
+  isHoverCssEnabled: PropTypes.bool,
 };
 Button.defaultProps = {
   className: null,
   isExternal: false,
   size: 'lg',
   type: 'primary',
+  disabled: false,
+  isHoverCssEnabled: true,
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/Button.jsx:71]: right, but why do we need buttons without hovering, anything clickable should have a hover state unless it's disabled, I think it just natural, don't you think?

File: common-util/Accordion.jsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+import { ChevronDown } from 'lucide-react';
+
+const transition = 'transition-all duration-300 ease-in-out';
+
+export const Accordion = ({ label, children }) => {
+  const [accordionOpen, setAccordionOpen] = useState(true);
+
+  return (
+    <div>
+      <button
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply common-util/Accordion.jsx:12]: sorry for what exactly

File: components/OperatePage/GettingStartedIsSimple.jsx [hunk 9]
```diff
@@ -0,0 +1,97 @@
+import React, { Fragment } from 'react';
+import Image from 'next/image';
+import { ArrowRight } from 'lucide-react';
+import { kebabCase } from 'lodash';
+
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Button } from 'components/Button';
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/OperatePage/GettingStartedIsSimple.jsx:6]: great idea 👍

File: common-util/Accordion.jsx (hunk 8 above)
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply common-util/Accordion.jsx:12]: I use this https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter 

but in this case it doesn't provide any suggestions, I think we may skip `aria-controls` as it's not required

File: components/OperatePage/Hero.jsx [hunk 10]
```diff
@@ -1,35 +1,64 @@
...
+    customClasses={`border-b ${SECTION_BOX_CLASS}`}
+    backgroundType="SUBTLE_GRADIENT"
+  >
+    <div className="grid max-w-screen-xl items-start mx-auto xl:gap-0 lg:px-12 lg:gap-8 lg:grid-cols-12 lg:items-center">
+      <div className="px-0 md:mb-12 lg:col-span-5 lg:px-5 lg:text-left">
+        <div className={`${TEXT_LIGHT_CLASS} mb-2 text-left`}>OLAS OPERATE</div>
+
+        <h2
+          className="leading-5 font-bold text-3xl sm:text-4xl lg:mb-8 lg:text-4xl xl:text-5xl xl:leading-5"
+          style={{ lineHeight: '120%' }}
```
Comment (Tanya-atatakai) [inline components/OperatePage/Hero.jsx:31]: can be `leading-[120%]` ? https://tailwindcss.com/docs/line-height#arbitrary-values

**Review by Tanya-atatakai — APPROVED** [review]
lgtm overall, there are several TODOs not related to updating links, can you pls check if they are still needed

File: components/OperatePage/InstallRunAnAgent.jsx [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+      },
+    );
+    return response.data;
+  } catch (error) {
+    console.error(error);
+    return null;
+  }
+}
+
+getLatestRelease();
```
Comment (Tanya-atatakai) [inline components/OperatePage/InstallRunAnAgent.jsx:116]: is it expected to have this call here?

File: components/OperatePage/Hero.jsx [hunk 12]
```diff
@@ -1,35 +1,61 @@
...
-          rel="noopener noreferrer"
+  <SectionWrapper
+    customClasses={`border-b ${SECTION_BOX_CLASS}`}
+    backgroundType="SUBTLE_GRADIENT"
+  >
+    <div className="grid max-w-screen-xl items-start mx-auto xl:gap-0 lg:px-12 lg:gap-8 lg:grid-cols-12 lg:items-center">
+      <div className="px-0 md:mb-12 lg:col-span-5 lg:px-5 lg:text-left">
+        <div className={`${TEXT_LIGHT_CLASS} mb-2 text-left`}>OLAS OPERATE</div>
+
+        <h2 className="font-bold !leading-[120%] text-3xl sm:text-4xl lg:mb-8 lg:text-4xl xl:text-5xl">
```
Comment (Tanya-atatakai) [inline components/OperatePage/Hero.jsx:29]: оО didn't know we can do that with "!"
> !leading-[120%]
cool

**Review by truemiller — APPROVED** [review]

**Pull request merged**